src/installing-deis/system-requirements.md

```diff
@@ -13,8 +13,34 @@ Note that these estimates are for Deis and Kubernetes only, and there should be
 
 Running smaller machines will likely result in increased system load and has been known to result in component failures and other problems.
 
-## Network
 
-Due to changes introduced in Docker 1.3.1 related to insecure Docker registries, the hosts running Deis must be able to communicate via a private network procured by kubernetes in the `10.0.0.0/8` private address space. This allows the docker daemon used by the controller to communicate with the registry. Ensure that your hosts have the following entries in their respective DOCKER_OPTS:
+# Kubernetes Requirements
 
-	EXTRA_DOCKER_OPTS="--insecure-registry 10.0.0.0/8"
+## Versions
+
+Deis workflow has been tested with the Kubernetes v1.1 release line. While Kubernetes 1.2 may work we haven't fully tested that release.
+
+## Daemon Sets
+
+The logging components require Kubernetes Daemon Sets API. DaemonSets are not enabled by default in the v1.1 release line, to enable these extensions follow the instructions found [here](http://kubernetes.io/v1.1/docs/api.html#enabling-resources-in-the-extensions-group). If you are running Kubernetes v1.2, DaemonSets are enabled by default.
+
+Specific steps to enable api extensions may vary based on your Kubernetes configuration. For example, to update a CoreOS Kubernetes cluster edit the API server unit file and add the following line to the `ExecStart` stanza: `--runtime_config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true`.
```

Comment:
There should be a comma after "cluster": "to update a CoreOS Kubernetes cluster, edit the API...".